install/kubernetes/helm/istio/charts/nodeagent/values.yaml

```diff
@@ -4,12 +4,18 @@
 enabled: false
 image: node-agent-k8s
 env:
-  # name of authentication provider.
+  # Name of authentication provider.
   CA_PROVIDER: ""
   # CA endpoint.
   CA_ADDR: ""  
-  # names of authentication provider's plugins.
+  # Names of authentication provider's plugins.
   PLUGINS: ""
+  # The agent will retrieve secrets from the API server with the frequency
+  # specified in this configuration.
+  SECRET_WATCHER_RESYNC_PERIOD: ""
```

Comment:
I think the "" is not a good idea. I know we have code in binary to handle if time.Duration parse error, we use as some hardcode value.

I would rather make this to match against the default values in code and then make hardcoded go in node agent simpler.